modules/labs-react/expandable/lib/ExpandableAvatar.tsx

```diff
@@ -0,0 +1,32 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import {jsx} from '@emotion/react';
+
+import React from 'react';
+
+import {createComponent, ExtractProps, useIsRTL} from '@workday/canvas-kit-react/common';
+import {Avatar} from '@workday/canvas-kit-react/avatar';
+import {space} from '@workday/canvas-kit-react/tokens';
+
+export interface ExpandableAvatarProps
+  extends Omit<
+    ExtractProps<typeof Avatar, never>,
+    keyof React.ButtonHTMLAttributes<HTMLButtonElement>
+  > {}
+
+export const ExpandableAvatar = createComponent('div')({
+  displayName: 'Expandable.Avatar',
+  Component: ({...elemProps}: ExpandableAvatarProps, ref, Element) => {
+    const margin = !useIsRTL() ? 'marginRight' : 'marginLeft';
```

Comment:
FYI, Our `styled` function from `common` handles left and right margins changes based on theme direction. It can be used instead of `useIsRTL` hook and `css` prop.